schemas/v3.0/schema.yaml

```diff
@@ -1,6 +1,6 @@
 id: https://spec.openapis.org/oas/3.0/schema/2021-09-28
 $schema: http://json-schema.org/draft-04/schema#
-description: Validation schema for OpenAPI Specification 3.0.X.
+description: description of OpenAPI v3.0.x documents, as defined by https://spec.openapis.org/oas/v3.0.3
```

Comment:
Just as a style thing, I usually prefer descriptions that are full sentences including starting with a capital letter. I don't feel to strongly about it if no one cares, but I wanted to make the suggestion anyway.

Reply:
pushed a revision.